using_images/xpaas_images/a_mq.adoc

```diff
@@ -5,4 +5,92 @@
 :icons:
 :experimental:
```

Comment:
Can you update the metadata and other bits to match the latest topic template here?

https://raw.githubusercontent.com/openshift/openshift-docs/master/contributing_to_docs/templates/topic_template.adoc

Of note, adding `:prewrap!` as well as including a TOC (`toc::[]`) and Overview section.